src/components/List/ListItem.tsx

```diff
@@ -47,25 +51,26 @@ class ListItem extends UIComponent<Extendable<IListItemProps>, any> {
     /** Toggle debug mode */
     debug: PropTypes.bool,
 
-    header: PropTypes.any,
     endMedia: PropTypes.any,
+    header: PropTypes.any,
     headerMedia: PropTypes.any,
 
     /** A list item can appear more important and draw the user's attention. */
     important: PropTypes.bool,
     media: PropTypes.any,
 
+    /** A list item can indicate that it is selected. */
+    selected: PropTypes.bool,
+
     /** A list item can indicate that it can be selected. */
     selection: PropTypes.bool,
-    truncateContent: PropTypes.bool,
-    truncateHeader: PropTypes.bool,
-
-    /** Accessibility behavior if overridden by the user. */
-    accessibility: PropTypes.func,
 
     /** Additional CSS styles to apply to the component instance.  */
     styles: PropTypes.oneOfType([PropTypes.object, PropTypes.func]),
 
+    truncateContent: PropTypes.bool,
```

Comment:
Please add description for these.

Reply:
I just rearranged the prop names in alphabetical order. These were not added by me in this PR. But you I can still go on and add those.

Reply:
@gopalgoel19, please, let me know if you need help with this

Reply:
those are good 👍